### DISCUSSION TITLE
DateRangePicker change format of calendar header

### DISCUSSION THREAD

**Discussion by lironezra** [rsuite/rsuite#1612] [Q&A]
Hello friends,

I'm using DateRangePicker component and I am importing it from like:
`import DateRangePicker from 'rsuite/lib/DateRangePicker/DateRangePicker';` (1)
I'm importing it in that way because my bundle size is much smaller from that way: 
 `import { DateRangePicker } from 'rsuite';` (2)

But there is an issue with the first way (1),  the calendar header (where we see the month and year) in ISO format like MM-YYYY
![image](https://user-images.githubusercontent.com/28096499/114504806-1dfaa800-9c38-11eb-893e-fc7a679782fb.png)
and I need the calendar month+year display like Feb, 2021.

I've tried  to use with `format` prop to change the format of the calendar but without success.

Someone face with same issue?

Thanks for help :)

**Comment by simonguo** [comment]
Repeated question https://github.com/rsuite/rsuite/issues/1627#issuecomment-834293075
